cishouseholds/derive.py

```diff
@@ -144,6 +144,31 @@ def assign_column_uniform_value(df: DataFrame, column_name_to_assign: str, unifo
     return df.withColumn(column_name_to_assign, F.lit(uniform_value))
 
 
+def assign_column_convert_to_date(df: DataFrame, column_name_to_assign: str, reference_column: str):
+    """
+    Assign a column with a TimeStamp to a DateType
+    From households_aggregate_processes.xlsx, derivation number 13.
+    Parameters
+    ----------
+    df
+    column_name_to_assign
+        Name of column to be assigned
+    reference_column
+        Column of TimeStamp type to be converted
```

Comment:
Minor suggestion - could indicate that this is the "Name of column..." for clarity

Reply:
Should be done.